utils/swift_build_support/swift_build_support/products/llvm.py

```diff
@@ -78,3 +86,592 @@ def _version_flags(self):
     @classmethod
     def get_dependencies(cls):
         return [cmark.CMark]
+
+    def llvm_c_flags(self, platform, arch):
+        # echo -n " $(common_cross_c_flags $1 ${LLVM_BUILD_TYPE})"
+        # if [[ $(is_cmake_debuginfo_build_type "${LLVM_BUILD_TYPE}") ]] ; then
+        #     if [[ $(is_llvm_lto_enabled) == "TRUE" ]] ; then
+        #         echo -n " -gline-tables-only"
+        #     else
+        #         echo -n " -g"
+        #     fi
+        # fi
+        result = self.common_cross_c_flags(platform, arch)
```

Comment:
Can you check if removing -target/-arch here breaks traditional cross compilation. I actually think it is probably ok to break those scenarios since we are going to be implementing the cross compilation in a correct cmake way. But we should know if we are going to break people.